app/src/main/java/com/kamron/pogoiv/pokeflycomponents/ocrhelper/OcrHelper.java

```diff
@@ -957,6 +959,13 @@ private String getPokemonIdentifierFromImg(Bitmap pokemonImage) {
     public ScanResult scanPokemon(@NonNull Bitmap pokemonImage, int trainerLevel) {
         ensureCorrectLevelArcSettings(trainerLevel); //todo, make it so it doesnt initiate on every scan?
         double estimatedPokemonLevel = getPokemonLevelFromImg(pokemonImage, trainerLevel);
+        LeveLRange estimatedPokemonLevelRange = new LeveLRange(estimatedPokemonLevel);
+        Optional<Integer> pokemonPowerUpStardustCost = Optional.absent();
+        //Optional<Integer> pokemonPowerUpStardustCost = getPokemonPowerUpStardustCostFromImg(pokemonImage);
```

Comment:
Is initializing the value to Optional.absent() intended?

Reply:
Yes, since the value is not currently used, I commented out the method since the OCR operation is somewhat expensive computational-wise.

Reply:
I supposed something like that! ACK!